### ISSUE TITLE
RDN training losses?

### ISSUE THREAD

**Issue by Sarah335** [idealo/image-super-resolution#221]
Wondering what loss functions the RDN model was trained on. For the 'noise-cancel' weights, it says that it was trained with adversarial and VGG feature losses - is this also true for 'psnr-large' and 'psnr-small' weights? or was the psnr loss the only metric used?
